src/Controllers/DatabaseController.js

```diff
@@ -399,6 +400,62 @@ DatabaseController.prototype.destroy = function(className, query, { acl } = {})
   });
 };
 
+const flattenUpdateOperatorsForCreate = object => {
+  for (let key in object) {
+    if (object[key] && object[key].__op) {
+      switch (object[key].__op) {
+        case 'Increment':
+          if (typeof object[key].amount !== 'number') {
+            throw new Parse.Error(Parse.Error.INVALID_JSON, 'objects to add must be an array');
+          }
+          object[key] = object[key].amount;
+          break;
+        case 'Add':
+          if (!(object[key].objects instanceof Array)) {
+            throw new Parse.Error(Parse.Error.INVALID_JSON, 'objects to add must be an array');
+          }
+          object[key] = object[key].objects;
+          break;
+        case 'AddUnique':
+          if (!(object[key].objects instanceof Array)) {
+            throw new Parse.Error(Parse.Error.INVALID_JSON, 'objects to add must be an array');
+          }
+          object[key] = object[key].objects;
+          break;
+        case 'Remove':
+          if (!(object[key].objects instanceof Array)) {
+            throw new Parse.Error(Parse.Error.INVALID_JSON, 'objects to add must be an array');
+          }
+          object[key] = []
+          break;
+        case 'Delete':
+          delete object[key];
+          break;
+        default:
+          throw new Parse.Error(Parse.Error.COMMAND_UNAVAILABLE, `The ${object[key].__op} operator is not supported yet.`);
+      }
+    }
+  }
+}
+
+const transformAuthData = (className, object, schema) => {
```

Comment:
...handled here, in Parse Server, not the DB adapter. The eventual plan is to somewhat duplicate the mongo format, so each type of auth data gets it's own column, like `_auth_data_facebook` and so-on.